README.md

```diff
@@ -83,37 +83,34 @@ You will need to use Node.js `Buffer` API compatible, if you are running inside
 const multihashing = require('multihashing-async')
 const buf = Buffer.from('beep boop')
 
-multihashing(buf, 'sha1', (err, multihash) => {
-  // by default calls back with a multihash.
-})
+const mh = await multihashing(buf, 'sha1')
 
 // Use `.digest(...)` if you want only the hash digest (drops the prefix indicating the hash type).
-multihashing.digest(buf, 'sha1', (err , digest) => {
-  // digest is the raw digest
-})
+const digest = await multihashing.digest(buf, 'sha1')
 
 // Use `.createHash(...)` for the raw hash functions
 const h = multihashing.createHash('sha1')
```

Comment:
Just a note (no need to change it). I think one letter variables are generally bad and as you already touch the surrounding code, I'd change `h` to `hash`.

Reply:
protip: make this suggestions and proposed changes using the new Github feature. It makes it easier for the owner to take them in.